docs/custom-images-contract.rst

```diff
@@ -95,10 +95,39 @@ Job arguments, in JSON format.
 .. code-block:: javascript
 
     {"key": "1111112/2/2", "project": 1111112, "version": "version1",
-    "spider": "spider-name", "spider_type": "auto", "tags": [],
-    "priority": 2, "scheduled_by": "user", "started_by": "admin",
+    "spider": "spider-name", "spider_type": "auto", "tags": ["tagA", "tagB"],
+    "priority": 2, "scheduled_by": "user", "started_by": "john",
     "pending_time": 1460374516193, "running_time": 1460374557448, ... }
 
+
+Some useful fields
+__________________
+
+============ ======================================================== =================================
+Field        Description                                              Example
+============ ======================================================== =================================
+key          Job key in format ``PROJECT_ID/SPIDER_ID/JOB_ID``        ``"1111112/2/2"``
+project      Integer project ID                                       ``1111112``
+spider       String spider name                                       ``"spider-name"``
+spider_args  Spider args dictionary                                   ``{"arg1":"val1"}``
+version      String project version used to run the job               ``"version1"``
+units        Amount of units used by the job                          ``1``
+priority     Job priority value                                       ``2``
+tags         List of string tags for the job                          ``["tagA", "tagB"]``
+state        Job current state name                                   ``"running"``
+pending_time UNIX timestamp when the job was added, in milliseconds   ``1460374516193``
+running_time UNIX timestamp when the job was started, in milliseconds ``1460374557448``
+auth         Job authentication string to access job data             ``""eyJ0e***.eyJhd***.9H5Oq***"``
```

Comment:
My point was that it's probably not very actionable for many/most users. It may trigger more questions than not documenting it (validity, where to use it, etc.)